testsuite/MDAnalysisTests/analysis/test_pca.py

```diff
@@ -56,6 +56,7 @@ def pca(u):
 
 @pytest.fixture(scope='module')
 def pca_aligned(u):
+    u.transfer_to_memory()
```

Comment:
I feel this is most efficiently done (and avoids numerous other "`transfer_to_memory()` calls) if `u` is loaded `in_memory=True` to start off with but not sure if there's a reason for this sequential nature of events

Reply:
If I remember correctly, in the original issue I couldn't get the code to work even if the test was isolated. It's possible I made a mistake somewhere, but definitely would be worth testing in isolation.